ext/opentelemetry-ext-http-requests/src/opentelemetry/ext/http_requests/__init__.py

```diff
@@ -103,12 +98,12 @@ def instrumented_request(self, method, url, *args, **kwargs):
 
```

Comment:
What if request returns no response?

Reply:
Is that even possible? I think if something goes wrong `requests.Session.request()` will throw an exception instead of returning nothing. Please correct me if I am missing something.

Reply:
You are correct.